src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,6 +124,12 @@ private int RunExecutable()
                 return result;
             }
 
+            // Add Nuget Packages Probing Path
+            var nugetPackagesRoot = _context.PackagesDirectory;
```

Comment:
To confirm, other commands like dotnet-compile or dotnet-build, etc, will be .NETCore apps and will be running against the SharedFramework when activated by dotnet.dll. Correct @brthor @piotrpMSFT ?